### PR TITLE
Mp tabbed interface

### PR DESCRIPTION
**Pull request by Ritumka**

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (jessjno) [inline app/controllers/users_controller.rb:1]: good job using the action call back!

File: app/views/photos/_photo.html.erb
Comment (jessjno) [inline app/views/photos/_photo.html.erb:1]: nice formatting

**Review by jessjno — APPROVED** [review]
looks good Margo!

File: app/views/users/_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="col md-6 offset-3">
```
Comment (armstrca) [inline app/views/users/_links.html.erb]: should be "col-md-6 offset-3"

**Review by armstrca — CHANGES_REQUESTED** [review]

File: app/views/users/_links.html.erb (hunk 1 above)
Comment (Ritumka) [reply app/views/users/_links.html.erb]: > should be "col-md-6 offset-3"

Got it! Thanks!

**Review by armstrca — APPROVED** [review]

**Pull request merged**